sync-orders-to-bigQuery/.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "@moltin"
```

Comment:
This repo contains a `.prettierrc`. This eslint can be removed and opt to use husky for formatting to follow the other examples.

Reply:
Removed